tasks/main.yml

```diff
@@ -37,9 +41,79 @@
     - mssql_upgrade | bool
     - mssql_version | int == 2017
 
+- name: Verify that EL < 8 is not used with mssql_ha_configure=true
+  assert:
+    that:
+      - ansible_distribution in ['CentOS', 'RedHat']
+      - ansible_distribution_version is version('8', '>')
+    fail_msg: >-
+      mssql_ha_configure=true does not support running against EL 7 hosts
```

Comment:
A very minor issue :)... Not "CentOS or RedHat newer than 8" includes not only `EL 7`, but also EL 6, Fedora, and other Linux distribution. I'd think they are also not supported, right? I thought it might be a bit confusing if the playbook failed when it's run against, e.g., Fedora and get `mssql_ha_configure=true does not support running against EL 7 hosts`.